camel/storages/vectordb_storages/qdrant.py

```diff
@@ -185,7 +185,9 @@ def _create_collection(
             VectorDistance.COSINE: Distance.COSINE,
             VectorDistance.EUCLIDEAN: Distance.EUCLID,
         }
-        self._client.recreate_collection(
+        # Since `recreate_collection` method will be removed in the future
+        # by Qdrant, `create_collection` is recommended instead.
```

Comment:
Could you please state the version of qdrant?

Reply:
Currently we don't know in which version Qdrant will remove `recreate_collection`, but we have warming from Qdrant